bedrock/base/templates/includes/protocol/navigation/nav-cta.html

```diff
@@ -4,6 +4,8 @@
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
+{% set utm_params = 'utm_source=www.firefox.com&utm_medium=referral&utm_campaign=nav&utm_content=cta' %}
```

Comment:
I'm assuming this is deliberte and `utm_params` are pulled from the context by a helper

Reply:
It's deliberate yeah